vmm_core/virt_mshv/src/lib.rs

```diff
@@ -1405,7 +1443,7 @@ impl virt::Processor for MshvProcessor<'_> {
 
 pub enum MshvRunVpError {
     #[error("/dev/mshv error")]
-    MsHv(#[from] vmm_sys_util::errno::Error),
+    MsHv(#[from] mshv_ioctls::MshvError),
```

Comment:
Same here as well, the underlying definition of the error has changed since this file was written.

Reply:
Replaced all MshvRunVpError instances into mshv_ioctls::MshvError in the latest iteration.